src/prefect/workers/base.py

```diff
@@ -872,6 +872,17 @@ async def _submit_scheduled_flow_runs(
                 run_logger.info(
                     f"Worker '{self.name}' submitting flow run '{flow_run.id}'"
                 )
+                if (
+                    get_current_settings().experiments.worker_logging_to_api_enabled
+                    and self.backend_id
+                ):
+                    worker_path = f"work-pools/work-pool/{self._work_pool_name}/worker/{self.backend_id}"
+                    base_url = get_current_settings().ui_url
```

Comment:
We have a [utility module for URLs](https://github.com/PrefectHQ/prefect/blob/main/src/prefect/utilities/urls.py) (`prefect.utitilies.urls`) where this URL generation would fit nicely.